lib/chainweb/client.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Kadena.Chainweb.Client do
+  @moduledoc """
+  Specifies the API for processing HTTP requests in the Kadena network.
+  """
+  alias Kadena.Chainweb.Client
+
+  @behaviour Client.Spec
+
+  @impl true
+  def request(method, path, chain_id \\ "0", headers \\ [], body \\ "", opts \\ []),
```

Comment:
@santiagoboterokommit IMO, the client should not manage the `chain_id`, this is something that should be contained in the request path .... in other words, the API endpoints are responsible for handling it